jsonnet/components/cluster-monitoring-operator.libsonnet

```diff
@@ -67,6 +67,17 @@ function(params) {
     data: {},
   },
 
+  federateClientCerts: {
```

Comment:
suggestion: 
It would we really nice if we started documenting what these secrets are getting used for. 

E.g.
```
// federate-client-certs secret is used by <X> to <achieve-what> 
```

Reply:
Sure.